src/test/java/com/dkohut/chatservice/ChatRoomDAOServiceTest.java

```diff
@@ -0,0 +1,105 @@
+package com.dkohut.chatservice;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.dkohut.chatservice.common.entity.ChatRoom;
+import com.dkohut.chatservice.common.interfaces.IChatRoomDAOService;
+
+/**
+ * 
+ * @author Dmytro Kohut
+ *
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ChatRoomDAOServiceTest {
```

Comment:
Everything is fine, just use it better for integration tests names like ChatRoomDAOIntegrationTests